domain/objects/obj_file.go

```diff
@@ -0,0 +1,53 @@
+package objects
+
+import (
+	"os"
+
+	"github.com/TudorHulban/GoLayouter/app/helpers/helpers"
+	"github.com/TudorHulban/GoLayouter/domain/interfaces"
+)
+
+type File struct {
+	Path    string
+	Content string
+}
+
+var _ interfaces.IFileOperations = &File{}
+
+func (f File) SetPath(path string) {
```

Comment:
you need a pointer receiver to mutate state

Reply:
why do we need this method?

Reply:
addressed in 3383f1a94dc26557ce25920ece2bf590e6b6dc84